snap/snapcraft.yaml

```diff
@@ -35,8 +36,19 @@ parts:
       mkdir -p $SNAPCRAFT_PART_INSTALL/opt
       cp -R $SNAPCRAFT_PART_BUILD $SNAPCRAFT_PART_INSTALL/opt/sabnzbd
       mkdir -p $SNAPCRAFT_PART_INSTALL/bin
-      echo "python3 \$SNAP/opt/sabnzbd/SABnzbd.py -f \$SNAP_COMMON" > $SNAPCRAFT_PART_INSTALL/bin/sabnzbd-wrapper
+      echo "\$SNAP/usr/bin/snapcraft-preload python3 \$SNAP/opt/sabnzbd/SABnzbd.py -f \$SNAP_COMMON" > $SNAPCRAFT_PART_INSTALL/bin/sabnzbd-wrapper
       chmod +x $SNAPCRAFT_PART_INSTALL/bin/sabnzbd-wrapper
+      ln -sf ../usr/lib/libsnapcraft-preload.so $SNAPCRAFT_PART_INSTALL/lib/libsnapcraft-preload.so
     prime: [-usr/bin/7z,-usr/bin/7za,-usr/bin/7zr]
     organize:
       usr/bin/unrar-nonfree: usr/bin/unrar
+  snapcraft-preload:
+    source: https://github.com/sergiusens/snapcraft-preload.git
+    plugin: cmake
+    cmake-parameters:
+      - -DCMAKE_INSTALL_PREFIX=/usr -DLIBPATH=/lib
```

Comment:
I'm a bit of a Linux/Cmake novice, but can't we change this parameter so the `ln -sf` a few lines up isn't needed anymore?

Reply:
I tried this a few times and spend a few hours, as others did: https://forum.snapcraft.io/t/snapcraft-preload-and-dev-shm-access/28200/ but wasnt able to find a solution (tried DCMAKE_INSTALL_PREFIX=/ but didnt work). Mabye someone else has a better way to handle this?